Unix/installbuilder/datafiles/Linux.data

```diff
@@ -214,5 +219,10 @@ if ${{PERFORMING_UPGRADE_NOT}}; then
         echo "Deleting omiusers group ..."
         groupdel omiusers
     fi
+    egrep -q "^omi:" /etc/group
+    if [ $? -eq 0 ]; then
+        echo "Deleting omi group ..."
+        groupdel omi
+    fi
 fi
```

Comment:
In this case, you're already in a loop to not delete the group during an upgrade, so this instance is okay.